BlogTemplate/Pages/Post.cshtml

```diff
@@ -28,13 +28,17 @@
                 <p name="Comment-Date" class="commentDate">@comment.PubDate.ToLocalTime().ToShortDateString()</p>

```

Comment:
It's not letting me comment on the right line, but on line 13, you shouldn't be showing the Edit Post link unless they are signed in. You already have the functionality that it redirects to login if they try going to this link, but it'd be even better to just hide this link until they're logged in.